### PR TITLE
fix(duckduckgo): change sitelinks class

### PR DESCRIPTION
**Pull request by kphrx**

### CODE REVIEW

**Comment by iorate** [comment]
Thank you for your contribution!

File: src/scripts/search-engines/duckduckgo.ts [hunk 1]
```diff
@@ -19,14 +19,14 @@ function defaultControlStyle(style?: CSSAttribute): (root: HTMLElement) => void
 export function getSerpHandler(): SerpHandler {
   return handleSerp({
     globalStyle: {
-      '[data-ub-blocked="visible"], [data-ub-blocked="visible"] + .results__sitelink--organics, [data-ub-blocked="visible"] .tile--img__sub':
+      '[data-ub-blocked="visible"], [data-ub-blocked="visible"] + .results__sitelinks--organics, [data-ub-blocked="visible"] .tile--img__sub':
```
Comment (kphrx) [inline src/scripts/search-engines/duckduckgo.ts:22]: sorry, typo for `.result__sitelinks--organics`

Comment (iorate) [reply src/scripts/search-engines/duckduckgo.ts:22]: Would you please create a new pull request?

Comment (kphrx) [reply src/scripts/search-engines/duckduckgo.ts:22]: opened #125

**Pull request merged**